pkg/oci/load_balancer_security_lists.go

```diff
@@ -43,65 +43,70 @@ const (
 	ProtocolUDP = 17
 )
 
+const (
+	// ModeManageAll denotes the management of security list rules for load
+	// balancer ingress/egress, health checkers, and worker ingress/egress.
+	ModeManageAll = "All"
```

Comment:
why not lowercase? Easier to remember.

Reply:
I was going with CamelCase to match the kube api. Happy with lower if you'd prefer.